x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/toolbar_additional_controls.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I can help with the tests for the files in the `data_table` folder since I broke them apart this way.... 😅

Reply:
It is okay.. I will be writing these soon.

Reply:
Danke!